src/sdk/bucketspace_fs.js

```diff
@@ -98,16 +99,41 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
             return account;
         } catch (err) {
             dbg.error('BucketSpaceFS.read_account_by_access_key: failed with error', err);
-            throw new RpcError('NO_SUCH_ACCOUNT', `Account with access_key not found`, err);
+            if (err.code === 'ENOENT') {
+                throw new RpcError('NO_SUCH_ACCOUNT', `Account with access_key not found.`, err);
+            }
+            throw new RpcError('NO_SUCH_ACCOUNT', err.message);
+        }
+    }
+
+    validate_account_schema(account) {
```

Comment:
hmm I think we want _ajv validation and not a custom validation, while creating a schema, validating etc, @liranmauda @guymguym WDYT ?

Reply:
same for the bucket validation

Reply:
adding here a reference for ajv validation we do in postgres client - https://github.com/noobaa/noobaa-core/blob/master/src/util/postgres_client.js#L1322